programs/zstdcli.c

```diff
@@ -1197,15 +1199,22 @@ int main(int const argCount, const char* argv[])
     /* IO Stream/File */
     FIO_setNotificationLevel(g_displayLevel);
     FIO_setPatchFromMode(prefs, patchFromDictFileName != NULL);
-    if (patchFromDictFileName != NULL) {
-        dictFileName = patchFromDictFileName;
-    }
     if (memLimit == 0) {
         if (compressionParams.windowLog == 0) {
             memLimit = (U32)1 << g_defaultMaxWindowLog;
         } else {
             memLimit = (U32)1 << (compressionParams.windowLog & 31);
     }   }
+    if (patchFromDictFileName != NULL) {
```

Comment:
We should check to ensure that we only have one file. Since we use `refPrefix()`, which is not sticky, only one file will get the dictionary.

Can you add a test to `playTests.sh` to make sure that `--patch-from` with more than one file errors?